globus_action_provider_tools/testing/fixtures.py

```diff
@@ -0,0 +1,42 @@
+from unittest.mock import patch
+
+import pytest
+
+from globus_action_provider_tools.flask.apt_blueprint import ActionProviderBlueprint
+from globus_action_provider_tools.testing.mocks import mock_authstate, mock_tokenchecker
+
+
+@pytest.fixture(scope="module")
+def apt_blueprint_noauth():
+    """
+    A fixture designed to mock an ActionProviderBlueprint instance's Globus
+    Auth integration. The fixture returns a function to which the instance
+    should be supplied as a parameter:
+
+    i.e. apt_blueprint_noauth(aptb)
+    """
+
+    def _apt_blueprint_noauth(aptb: ActionProviderBlueprint):
+        # Manually remove the function that creates the internal token_checker
+        for f in aptb.deferred_functions:
+            if f.__name__ == "_create_token_checker":
```

Comment:
Given how this is mucking with the internals, really good to provide the function for people as we wouldn't want them to have to figure this out.